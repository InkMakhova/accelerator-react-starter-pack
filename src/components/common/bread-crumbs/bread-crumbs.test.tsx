import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router, Route } from 'react-router-dom';
import BreadCrumbs from './bread-crumbs';

it('should render BreadCrumbs correctly', () => {
  const {container} = render(
    <Router>
      <Route>
        <BreadCrumbs />
      </Route>
    </Router>);
  expect(container).toMatchSnapshot();
});
