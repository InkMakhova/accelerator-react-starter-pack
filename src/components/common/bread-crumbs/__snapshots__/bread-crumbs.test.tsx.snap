// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BreadCrumbs correctly 1`] = `
<div>
  <ul
    class="breadcrumbs page-content__breadcrumbs"
  >
    <li
      class="breadcrumbs__item"
    >
      <a
        class="link"
        href="/"
      >
        Главная
      </a>
    </li>
    <li
      class="breadcrumbs__item"
    >
      <a
        class="link"
        href="/catalog"
      >
        Каталог
      </a>
    </li>
  </ul>
</div>
`;
